cmd/sqlcmd/sqlcmd.go

```diff
@@ -117,25 +117,109 @@ func (a SQLCmdArguments) authenticationMethod(hasPassword bool) string {
 }
 
 func Execute(version string) {
-	ctx := kong.Parse(&args, kong.NoDefaultHelp())
-	if args.Version {
-		ctx.Printf("%v", version)
-		os.Exit(0)
+	// Conbra Tryout #2 : This has version and help flags. But they are also getting stucked inside run function without error
+	rootCmd := &cobra.Command{
+		Use:   "testCommand",
+		Short: "A brief description of your command",
+		Run: func(cmd *cobra.Command, argss []string) {
+			vars := sqlcmd.InitializeVariables(!args.DisableCmdAndWarn)
+			setVars(vars, &args)
+
+			if args.Version {
+				fmt.Printf("%v\n", version)
+				os.Exit(0)
+			}
+
+			if args.Help {
+				fmt.Print(cmd.UsageString())
+				os.Exit(0)
+			}
+
+			exitCode, err := run(vars, &args)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+
+			fmt.Printf("after exit exitCode: %d\n", exitCode)
+			os.Exit(exitCode)
+			fmt.Printf("after execute Function")
+		},
 	}
-	if args.Help {
-		_ = ctx.PrintUsage(false)
-		os.Exit(0)
+
+	//var version string
+
+	rootCmd.PersistentFlags().BoolVarP(&args.Version, "version", "v", false, "Print version information and exit")
```

Comment:
> "v" [](http://example.com/codeflow?start=61&length=4)

there's no short flag for this parameter

Reply:
Removed short flag.